AAI/AAIConnectProfile.md

```diff
@@ -747,6 +832,123 @@ See the [GA4GH Passport
 Claim Format](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md#passport-claim-format)
 for more details.
 
+### Broker Passport Endpoint
+
+This endpoint is only REQUIRED if the Broker has published the endpoint's location
+via OIDC Discovery. See the [Conformance for Brokers](conformance-for-brokers]
+section for details.
+
+#### Passport Endpoint Request
+
+```
+POST <ga4gh-passport-endpoint>
+Authorization: Bearer <passport-scoped-access-token>
+Content-Type: application/json
+
+{
+  "ga4gh_passport:self_contained_v1": "<self-contained-passport-jws>",
+  "targets": {
```

Comment:
What is the benefit of allowing several targets in one request vs. sending several requests each with one target?